### PR TITLE
Fixed aaveV2 on avalanche + minor updates

### PR DESCRIPTION
**Pull request by pradyuman-verma**
- [x] Fixes aaveV2 on avalanche (chainlink price feed oracle fixed)
- [x] minor updates
- [x] Deploy and comment the address

### CODE REVIEW

**Comment by pradyuman-verma** [comment]
Avalanche aaveV2 test: 
image

File: contracts/protocols/polygon/aave_v2/main.sol [hunk 1]
```diff
@@ -53,6 +53,11 @@ contract Resolver is AaveHelpers {
         AaveAddressProvider addrProvider = AaveAddressProvider(getAaveAddressProvider());
         data = getList(AaveLendingPool(addrProvider.getLendingPool()));
     }
+
+    function getPrice(address[] memory tokens) public view returns (TokenPrice[] memory tokenPrices, uint256 ethPrice) {
```
Comment (pradyuman-verma) [inline contracts/protocols/polygon/aave_v2/main.sol:57]: @KaymasJain Added this function to get priceFeed for testing purposes !!
Is it okay?

**Comment by pradyuman-verma** [comment]
Polygon aaveV2 test:
image

File: contracts/protocols/polygon/aave_v2/main.sol (hunk 1 above)
Comment (KaymasJain) [reply contracts/protocols/polygon/aave_v2/main.sol:57]: Sure!

**Comment by pradyuman-verma** [comment]
aaveV2 deployed on polygon - [0x3BCA6531eDb2b41319fC56bA3c142367F83B65EE](https://polygonscan.com/address/0x3BCA6531eDb2b41319fC56bA3c142367F83B65EE)

**Comment by pradyuman-verma** [comment]
aaveV2 deployed on avalanche - [0x313FE505ad3ead0D35dD5d6687FB9C6B2469Db3d](https://snowtrace.io/address/0x313FE505ad3ead0D35dD5d6687FB9C6B2469Db3d)

**Comment by pradyuman-verma** [comment]
Deployed addresses can be found above.

**Pull request merged**